rlselect.py

```diff
@@ -99,6 +99,19 @@ def match(line):
             line = line.lower()
         marks = set()
         for term, term_len in terms:
+            # Negative matching..
+            if term[0] == '!':
+                # A single exclamation char always matches
+                if term == '!':
+                    continue
+                # A double exclamation string means match for single exclamation char
+                elif term == '!!':
+                    term = '!'
```

Comment:
Check if term starts with double !!, then remove one so that it can match "!any"?

Reply:
Yes, that's how it works.

Reply:
Yes, of course, you are absolutely right.